config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/customers/:id", to: "customers#show"
+  patch "/customers/:id", to: "customers#update"
+
+  post "/customer_items", to: "customer_items#create"
```

Comment:
yes, this route works just fine. You could also have done something like `post "/customers/:id/items", to: "customer_items#create"` that way you'd already have the customer_id in your params hash. either way is fine.